### ISSUE TITLE
text detection

### ISSUE THREAD

**Issue by pasha76** [da03/Attention-OCR#63]
How do you guys detect the texts to recognize them with this code? Any suggestion? I guess I have to crop the texts in a scene and feed them to this code, right?

Thanks

**Comment by MBleeker** [comment]
Yes, there are a lot of possible options for this. The option you mentioned is one, but you could also try to combine the detection with the recognition in one model. But that is an open research problem at the moment :)

Cheers
